api-report/container-runtime.api.md

```diff
@@ -235,7 +235,7 @@ export class FluidDataStoreRegistry implements IFluidDataStoreRegistry {
 export const gcBlobPrefix = "__gc";
 
 // @public (undocumented)
-export const gcTreeKey = "gc";
```

Comment:
This renaming isn't necessary for the fix, right? Do you want to do that as a separate change?

Reply:
sure, I could do that. Since it's a small change I did it here.